packages/salesforcedx-sobjects-faux-generator/src/describe/sObjectDescribe.ts

```diff
@@ -244,7 +245,8 @@ export class SObjectDescribe {
         'Content-Type': 'application/json',
         Accept: 'application/json',
         Authorization: `OAuth ${this.accessToken}`,
-        'User-Agent': 'salesforcedx-extension'
+        'User-Agent': 'salesforcedx-extension',
+        'Sforce-Call-Options': `client=${CLIENT_ID}`
```

Comment:
This will add the logging for direct REST calls. I followed Jonathan's example from the apex-debugger package. One thing I would like to note here is that I noticed that there are no outside references to describeSObject (this method) other than a test for the method. If we're not using it, should we remove it?

/cc @GaryFNC

Reply:
I left describeSObject  there (it originally preceded the batch calls) because there is a good chance it will be used when processing the XML for custom objects.     But it could be removed and added back later if needed.  OK either way